### PR TITLE
add conan training series post

### PR DESCRIPTION
**Pull request by solvingj**
Ping @memsharded for review

### CODE REVIEW

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 1]
```diff
@@ -0,0 +1,103 @@
+---
+layout: post 
+comments: false 
+title: "Conan Training Series on JFrog Academy"
+---
+
+Today we are very exicted to announce a new milestone in the Conan project. We
+have now published our very own self-paced interactive training courses online,
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:8]: Add "free", probably also in the title of the post.

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+free to anyone who is interested in learning Conan. The courses are hosted on
+JFrog's very own learning platform: JFrog Academy, which also features a wide
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:10]: Add here the "why". You can say that the online trainings were very succesful, with very high demand, and when we did a break for the summer we had a waiting list. You can include the data:
- 20 sessions in 2 different timezones
- 460 trainees total
- Very good feedback: 70% trainees rated it "excellent", 25% rated it "very good"

And we thought that moving online self-paced was the right thing for scale, so everybody can enjoy, and they can also do it without being restricted to timezones or availabilty.

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+variety of courses on other related topics.  
+
+## Learning Format
+
+The courses are adapted from the interactive training sessions we've held in the
+past at CPPCon and online over the past two years. They feature interactive
+exercises which walk users through the running of commands, exploring and
+editing of importantC Conan-related files and directories, and quizzes to invoke
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:18]:
```suggestion
editing of important Conan-related files and directories, and quizzes to invoke
```

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+is intended for developers with little-to-no Conan experience and will take you
+through a series of interactive exercises. Along the way, we'll explain
+fundamental Conan concepts such as "Package ID",  "generators",  "settings",
+"options", and "profiles", as well as the basic anatomy of the "Conanfile".  By
+the end of the training you will have enough experience to start working with
+Conan as part of your daily development process.  
+
+## Conan Advanced
+
+The second course in the series is called "Conan Advanced". This course is
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:35]: Will also be available now, right? Can we say it?

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 5]
```diff
@@ -0,0 +1,103 @@
...
+versioning topics such as the use of semantic versioning, version ranges,
+Conan's built-in revisioning system, and Conan's Lockfile feature. It will also
+include a deeper dive into the topics of Conan configuration and Package ID. By
+the end of the training, you will have a deeper understanding of how these
+advanced features are intended to be used, and how they can solve some of the
+more challenging real-world scenarios that emerge when using Conan at Scale.  
+
+## Conan CI/CD
+
+The third course will be next on the list and will be titled "Conan CI/CD".
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:52]: "Will be next on the list" clearly means it is not available yet, but soon will be? Maybe make this meaning more explicit

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+comments: false 
+title: "Conan Training Series on JFrog Academy"
+---
+
+Today we are very exicted to announce a new milestone in the Conan project. We
+have now published our very own self-paced interactive training courses online,
+free to anyone who is interested in learning Conan. The courses are hosted on
+JFrog's very own learning platform: JFrog Academy, which also features a wide
+variety of courses on other related topics.  
+
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:12]: Let's add a screen capture of the academy, with the available courses, and maybe at least a screen capture of one video, one slide with your face if possible.

I like the idea of having in this post the intro video embedded.

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 7]
```diff
@@ -0,0 +1,103 @@
...
+[Conan Series in JFrog Academy](https://academy.jfrog.com/series/conan)
+
+This page shows all the currently available Conan courses.  With that said, we
+encourage everyone to checkout the entire catalog of courses in JFrog
+artifactory.  
+
+[JFrog Academy Course Catalog](https://academy.jfrog.com/series/course-catalog)
+
+Access to the courses requires a signup, but it's completely free to register.
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:84]: Maybe add here some brief explanation that the hands-on exercises will use docker in the client side to achieve unified experience, but make sure to tell that it is not necessary to know docker at all, only to install docker in your computer, any OS including Windows, and instructions are included in the video course itself.

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 8]
```diff
@@ -0,0 +1,103 @@
...
+existing courses, or want to suggest a new course, please open an issue under
+the following Github repository:
+
+[Conan Training Repository on Github](https://github.com/conan-io/training)
+
+This repository contains all the content for all the exercises in both the
+Essentials and Advanced courses, and so it's also the best place to report
+any sort of feedback.  
+
+As you complete the coureses, if you enjoyed them and want to help spread the
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:99]:
```suggestion
As you complete the courses, if you enjoyed them and want to help spread the
```

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 9]
```diff
@@ -0,0 +1,103 @@
...
+word, there are links to promote them on Twitter and LinkedIn.
+
+Stay tuned for more updates about new coureses and updates to existing coureses
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:102]:
```suggestion
Stay tuned for more updates about new coureses and updates to existing courses
```

File: _posts/2020-09-14-New-conan-training-series.markdown [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+encourage everyone to checkout the entire catalog of courses in JFrog
+artifactory.  
+
+[JFrog Academy Course Catalog](https://academy.jfrog.com/series/course-catalog)
+
+Access to the courses requires a signup, but it's completely free to register.
+
+## Feedback and Improvement
+
+We already have a lot of ideas for improving all of our courses over time and
```
Comment (memsharded) [inline _posts/2020-09-14-New-conan-training-series.markdown:88]: Maybe add to this section title: "Support, feedback and improvement"

Then briefly explain that we will be there to help them during the training, if they have any issues, they can file a ticket at the training repo to ask for help.

**Review by memsharded — COMMENTED** [review]
Almost there :)

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 4 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown:35]: yes

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 10 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: done in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 9 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: done in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 8 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: done in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 7 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: done in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 6 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: done in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 5 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: fixed in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 3 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: fixed in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 2 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: added in 6514c39

File: _posts/2020-09-14-New-conan-training-series.markdown (hunk 1 above)
Comment (solvingj) [reply _posts/2020-09-14-New-conan-training-series.markdown]: added in 6514c39

**Review by memsharded — APPROVED** [review]

**Pull request merged**